### PR TITLE
Update yaml files with scope and permission required for each API

### PR DESCRIPTION
**Pull request by DinikaSen**
## Purpose
Update yaml files with scope and permission required for each API

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml [hunk 1]
```diff
@@ -96,13 +103,14 @@ paths:
     get:
       tags:
         - Email Template Types
-      summary: Retrieve the email template type corresponds to the template type id.
+      summary: Retrieves the email template type corresponds to the template type id.
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml]: corresponding to the

File: components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml [hunk 2]
```diff
@@ -129,15 +137,16 @@ paths:
...
-      summary: Add a new email template to an existing email template type.
+      summary: Adds a new email template to an existing email template type.
       operationId: addEmailTemplate
-      description: |
-        Adds a new email template an existing email template type in the system.
-        The locale of the new email template should not already exists in the
-        respective email template type.
+      description: >
+        Adds a new email template to an existing email template type in the system.
+        Another email template with the same locale should not already exists in the
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml]: should not exist in the

File: components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml [hunk 3]
```diff
@@ -207,12 +218,14 @@ paths:
         - Email Template Types
       summary: Removes an email template type.
       operationId: deleteEmailTemplateType
-      description: |
+      description: >
         Removes an existing email template type with all it's email templates
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml]: with all its email templates

File: components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml [hunk 4]
```diff
@@ -269,10 +284,10 @@ paths:
       summary: Retrieves a single email template.
       operationId: getEmailTemplate
       description: |
-        Retrieves the email template that matches to the template-type-id and the template-id
+        Retrieves the email template that matches to the template-type-id and the template-id.<br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.email.template/org.wso2.carbon.identity.rest.api.server.email.template.v1/src/main/resources/email-template.yml]: that matches the template-type-id and the template-id.

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 5]
```diff
@@ -75,7 +79,11 @@ paths:
       summary: |
         Add a new Identity Provider
       description: |
-        This API provides the capability to create a new Identity Provider
+        This API provides the capability to create a new Identity Provider <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: identity provider

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 6]
```diff
@@ -141,7 +149,11 @@ paths:
         Metadata about supported federated authenticators of identity providers
       description: >-
         This API provides the list of supported federated authenticators for an
-        identity provider in Identity Server
+        identity provider in Identity Server <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: the identity server

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 7]
```diff
@@ -182,7 +194,11 @@ paths:
         Metadata about a supported federated authenticator
       description: >
         'This API provides the details of a single supported federated
-        authenticator for an identity provider in Identity Server'
+        authenticator for an identity provider in Identity Server' <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: in the identity server

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 8]
```diff
@@ -229,7 +245,11 @@ paths:
         providers in Identity Server
       description: >-
         This API provides the list of supported federated authenticators for an
-        IDP in Identity Server
+        IDP in Identity Server <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: in the identity server.

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 9]
```diff
@@ -270,7 +290,11 @@ paths:
         Metadata about supported outbound provisioning connectors
       description: >-
         This API provides the details of a single supported outbound
-        provisioning connectors for an IDP in Identity Server
+        provisioning connectors for an IDP in Identity Server <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: in the identity server

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 10]
```diff
@@ -568,7 +612,11 @@ paths:
         authenticator id
       description: >
         This API provides the capability to update an identity provider's
-        federated authenticator config by specifying authenticator id
+        federated authenticator config by specifying authenticator id <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: specifying the authenticator id.

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 11]
```diff
@@ -633,7 +681,11 @@ paths:
       description: >
         'This API provides a list of available provisioning entities of an
         identity provider. This includes just-in-time provisioning config and
-        outbound provisioning connectors'
+        outbound provisioning connectors' <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: Remove single quotes in before 'This API and after connectors'

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 12]
```diff
@@ -992,7 +1068,11 @@ paths:
         Update the role config of an identity provider
       description: >
         This API provides the capability to update the role config of an
-        identity provider by specifying identity provider id
+        identity provider by specifying identity provider id <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: the identity provider id

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 13]
```diff
@@ -1050,7 +1130,11 @@ paths:
         Just-In-Time provisioning config of an identity provider
       description: >
         This API retrives the Just-In-Time provisioning config of an identity
-        provider by specifying identity provider id
+        provider by specifying identity provider id <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: the identity provider id

File: components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml [hunk 14]
```diff
@@ -1155,7 +1243,11 @@ paths:
       summary: |
         Connected applications of an identity provider
       description: >
-        This API provides the list of applications which are using this identity provider for federated/provisioning.
+        This API provides the list of applications which are using this identity provider for federated/provisioning. <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml]: federation/provisioning

File: components/org.wso2.carbon.identity.api.server.keystore.management/org.wso2.carbon.identity.api.server.keystore.management.v1/src/main/resources/keystore.yaml [hunk 15]
```diff
@@ -52,7 +56,11 @@ paths:
         - Certificates
       summary: uploads the certificate with the given alias
       operationId: uploadCertificate
-      description: This REST API can be used to upload the certifate to the tenant keystore. This API is not supported for super tenant. 
+      description: This REST API can be used to upload the certifate to the tenant keystore. This API is not supported for super tenant. <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.keystore.management/org.wso2.carbon.identity.api.server.keystore.management.v1/src/main/resources/keystore.yaml]: certificate

File: components/org.wso2.carbon.identity.api.server.oidc.scope.management/org.wso2.carbon.identity.api.server.oidc.scope.management.v1/src/main/resources/oidc-scope-management.yaml [hunk 16]
```diff
@@ -153,6 +171,11 @@ paths:
         - OIDC Scope Endpoint
       summary: Delete an OIDC Scope
       operationId: deleteScope
+      description: This API can be used tp delete an existing OIDC scope <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.oidc.scope.management/org.wso2.carbon.identity.api.server.oidc.scope.management.v1/src/main/resources/oidc-scope-management.yaml]: used to

File: components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml [hunk 17]
```diff
@@ -23,8 +23,12 @@ paths:
         - Script Libraries
       summary: |
         Add a new script library.
-      description: >
-        This API provides the capability to create a new Script library.
+      description: |
+        This API provides the capability to create a new Script library. <br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml]: script

File: components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml [hunk 18]
```diff
@@ -107,8 +115,12 @@ paths:
         - Script Libraries
       summary: |
         Get script library.
-      description: >
-        This API provides the capability to retrive the script library details by using its name.
+      description: |
+        This API provides the capability to retrive the details of the script library using its name.<br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml]: retrieve

File: components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml [hunk 19]
```diff
@@ -238,8 +258,12 @@ paths:
         - Script Libraries
       summary: |
         Get the content of the script library.
-      description: >
-        This API provides the capability to retrive the content of script library by using its name.
+      description: |
+        This API provides the capability to retrive the content of script library using its name.<br>
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/resources/scriptLibrary.yaml]: retrieve

File: components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml [hunk 20]
```diff
@@ -126,8 +137,11 @@ paths:
         This API provides the capability to delete a secondary user store
         matching to the given user store domain id.
```
Comment (gomathyk) [inline components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml]: matching the given

**Comment by DMHP** [comment]
Please resolve the conflicts and execute the integration tests after locally merging this.

**Comment by DinikaSen** [comment]
Closing because these changes are already applied from a differet PR

**Pull request closed without merging**